superbuild/catch2/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+# Use CATCH2_URL to specify the location of the git repo. Use
+# CATCH2_TAG to specify the commit.
+
+enable_language(CXX)
+
+# Handle the clone mechanism. First URL
+option(CATCH2_CLONE_VIA_SSH
+  "Clone CATCH2 using SSH instead of HTTPS" ${LBANN_SB_CLONE_VIA_SSH})
+
+if (CATCH2_CLONE_VIA_SSH)
+  set(CATCH2_URL git@github.com:catchorg/catch2.git
+    CACHE STRING "The URL from which to clone CATCH2")
+else ()
+  set(CATCH2_URL "https://github.com/catchorg/catch2.git"
+    CACHE STRING "The URL from which to clone CATCH2")
+endif ()
+
+# ... then the tag.
+set(CATCH2_TAG "master"
```

Comment:
Would it be better pull a release instead of master? It would be bad if our testing framework broke...

Reply:
Done.